build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1'
     classpath "org.jetbrains.dokka:dokka-android-gradle-plugin:$dokka_version"
-    classpath 'com.facebook.testing.screenshot:plugin:0.13.0'
+    classpath 'com.facebook.testing.screenshot:plugin:0.14.0'
```

Comment:
Were there any issues with upgrading the ST library? There were some in the past.

Reply:
no issues at all, no. it'll actually be needed when we upgrade to gradle 7.0